test/ja_serializer/builder/relationship_test.exs

```diff
@@ -22,7 +22,13 @@ defmodule JaSerializer.Builder.RelationshipTest do
 
   defmodule FooSerializer do
     use JaSerializer
-    has_many :bars, type: "bar"
+    has_many :bars,
+             type: "bar",
+             relationship_links: [
```

Comment:
I kind of feel like the "relationship" is a bit of a given here, what do you think about just "links"?